README.md

```diff
@@ -1,6 +1,6 @@
 # bs-use-fetch
 
-A ReasonReact useFetch hook. It:
+A React useFetch hook. It:
```

Comment:
I think this is one place where search-and-replace didn't quite work :)

```suggestion
- A React useFetch hook. It:
+ A useFetch hook for reason-react. It:
```

Feel free to reword :)

Reply:
Good eye. At the very least, its being used as a learning tool!

Update with `a Reason/Rescript React useFetch hook` figuring the current state of the ecosystem. Let me know. 


Thanks again.